### ISSUE TITLE
Credibility of MTAG loci

### ISSUE THREAD

**Issue by mpx353** [omeed-maghzian/mtag#61]
Dear Omeed, 

When running MTAG on the GWAS results of two related traits (UKB data, phenotypic correlation 0.6), with sample overlap, I picked up some loci that were not significant in the GWAS. However, I also noticed that for some variants that I did pick up in the GWAS, the mtag_pval is almost flat in the MTAG results. For example:

GWAS results (SNP | CHR | BP | EA | EAF | **Pval** | N | beta | SE)

rs12940610 | 17 | 64312463 | A | 0.422 | **7.40E-12** | 51399 | 0.0391154 | 0.00571079

Whereas in the MTAG results, the same SNP gives (**Pval** | N | beta | SE):
**2.26E-03** | 51399 | 0.018 | 0.006

Previous threads here have discussed the role of the mean GWAS chi^2 to assess the reliability of MTAG results. I had a look at the log output, and indeed the mean GWAS chi^2 is below 1.1 for both traits.
 
  Trait                 N (max)  N (mean)  # SNPs used  GWAS mean chi^2  MTAG mean chi^2  GWAS equiv. (max) N
1  ..._MTAG_010518.txt  51884    50766     7168585      1.085            1.085            51993              
2  ..._MTAG_010518.txt  51503    50394     7168585      1.049            1.055            57041

I was just wondering, if you could give any advice here on how to proceed and whether you still would recommend to use MTAG in this case. From the paper and discussion here, it is clear that the chi^2 is an important indicator for the reliability of the MTAG analysis. 
Also, could you perhaps speculate how it can happen that two moderate related traits derived from the same cohort cannot be used for MTAG? Could this also indicate a problem with the original GWAS or phenotype? The GWAS used for input had been filtered, such that MAF>0.1 and the INFO score>0.3. 

Many thanks,
Stefan

**Comment by paturley** [comment]
Hi Stefan,

Apologies for the delay in replying. If a SNP is significant before
applying MTAG but not afterwards, it is likely because the beta estimate
for the correlated trait has an opposite sign or an estimate that is much
smaller in magnitude.

Your case is sort of on the edge of what I would think is fine for MTAG.
Have you run the maxFDR analysis? I like to do that to get a sense of how
reliable the MTAG hits will be.

I don't understand your last question. Can you clarify?
You

**Comment by mpx353** [comment]
Many thanks for your reply. I had not performed the maxFDR yet, doing this now. 

My last question was about the mean chi^2 value. I was just wondering why my mean chi^2 values are low. For example, do opposite beta estimates between correlated trait contribute to a low mean chi^2 value?

Thanks!

**Comment by paturley** [comment]
The mean chi2 scales with the sample size and heritability of the traits
you use. With just 50k samples in your data, your mean chi2 stats seem
about right to me. The increase in mean chi2 for the MTAG summary stats
depends on the sample overlap, the genetic correlation, and the phenotypic
correlation. Since you have total overlap, I would expect to see a slight
gain in the mean chi2 but not a large one, which it looks like is the case.

Best,
Patrick

**Comment by mpx353** [comment]
Thanks Patrick. I just got the max FDR results:

Maximum FDR
Max FDR of Trait 1: 0.0985271605346 at probs = [0.  0.5 0.  0.5]
Max FDR of Trait 2: 0.212188308953 at probs = [0.  0.  0.6 0.4]

Especially for trait 2, the FDR is very high, which may explain our results and probably reflects the fact that our sample size is low?

**Comment by paturley** [comment]
Yeah. You might consider calculating the maxFDR for each trait run
separately as a baseline. The maxFDR will likely be pretty high then as
well.

**Comment by mpx353** [comment]
Just for your interest: Since the maxFDR was quite high, I was interested in the actual overlap of significant variants in the GWAS vs. MTAG. See below the results:

Trait 1: (maxFDR ~ 0.10)
No. of SNPs GWS in both GWAS and MTAG: 1484 (~0.017% of total number of SNPs)
No. of SNPs GWS in GWAS but not MTAG: 3
No. of SNPs GWS in MTAG but not GWAS: 31 (only ~2% increase wr. to GWAS findings)

Trait 2: (maxFDR ~ 0.21)
Significant in both GWAS and MTAG: 98 (=0.001%)
Significant in GWAS but not MTAG: 36
Significant in MTAG but not GWAS: 146 (~9% increase wr. to GWAS findings)

For trait 2, there was a higher increase of significant variants compared to trait 1, which is in line with the higher maxFDR for trait 2. However, the increase is lower than what I would have expected from the maxFDR results. Theoretical, I would have expected to find ~21% more variants with MTAG for trait 2, but I suppose that's why we call it the maxFDR and not the meanFDR?

Thanks again for your help with this issue!

**Comment by paturley** [comment]
No problem. Glad to help.

**Comment by mpx353** [comment]
To finish/summarise this thread: do you think one could use MTAG in this case (provided awareness of the potential limitations implicated by the high FDR) or would you recommend to not use MTAG in this case? I couldn't find a clear threshold for the FDR, but I suspect 5% would be a good value? Thanks for your advice.

**Comment by paturley** [comment]
I don't know that there is a clear threshold. It really depends on the
application. Have you run maxFDR on each of the sets of summary statistics
separately? Sometimes that's a good baseline to compare how good the MTAG
summary statistics are relative to the GWAS summary stats.

Patrick

**Comment by Xuemin-Wang** [comment]
Hi Patrick,

You mentioned that if a SNP is significant before applying MTAG but not afterwards, it is likely because the beta estimate for the correlated trait has an opposite sign or an estimate that is much smaller in magnitude. Does this mean that negatively correlated traits cannot included into the MTAG analysis? Will it unlikely result in the detection of significant gwas loci by MTAG?

Many thanks,

**Comment by paturley** [comment]
MTAG will work fine if traits have a negative genetic correlation as long
as the assumptions of MTAG are met. More precisely, I should have said that
SNPs may fall from significance if the other trait has an association that
has an opposite sign than expected. So if the traits are negatively
correlated in general, but some SNPs have estimates that have the same sign
of association, a SNP that was previously statistically significant may
lose statistical significance. But this is not the only reason that a SNP
may lose statistical significance. Sometimes that occurs by pure chance due
to sampling variation even if all assumptions are met.
